tiled-maps/scripts/generateMapConfig.mjs

```diff
@@ -210,7 +212,7 @@ function combineSingleLayer(childMaps, layerName) {
       const mapName = `${base}-0${idx}`;
       const targetLayer = childMaps[mapName].layers
         .filter((layer) => layer.name.toLowerCase() === layerName)[0];
-
+      if (targetLayer === undefined) return null;
```

Comment:
The data in combinedLayer might be incorrect, because it's missing the data here in this `mapName`, but it might exist in other `mapName`.